src/SSEConfig.cpp

```diff
@@ -70,7 +70,9 @@ bool SSEConfig::load(const char *file) {
   // Read config file.
   try {
     boost::property_tree::read_json(file, pt);
-  } catch (...) {}
+  } catch (boost::property_tree::ptree_error& e) {
+    LOG(WARNING) << "Error during loading config file: " << e.what();
```

Comment:
Perhaps log `FATAL` instead of `WARNING`? It shouldn't be able to start up at all without a proper config.

Reply:
Thanks for noticing, this is absolutely a check we should have. I agree with @sgulseth regarding that it should be a FATAL and not a warning. Add in that and I will merge the PR!

Reply:
Ok, I've changed it to FATAL and now the app doesn't start when config is not parsed or not present. Thanks for your feedback.